pegasus/sites.v3/code.org/public/lms.haml

```diff
@@ -0,0 +1,67 @@
+---
+title: LMS
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/lms-styles.css", rel: "stylesheet"}
+
+%section.top
+  .wrapper
+    .text-wrapper.col-50.pad
+      %h1=hoc_s("lms_page.heading")
+      %p.body-two
+        =hoc_s("lms_page.top_desc")
+      %a.link-button{href: "#"}
```

Comment:
Should this be linking to the top of the page or somewhere else?

Reply:
ahh I forgot to go back and add the section! Fixed and thank you